backend/src/test/java/com/bakdata/conquery/models/auth/LocalAuthRealmTest.java

```diff
@@ -0,0 +1,104 @@
+package com.bakdata.conquery.models.auth;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.File;
+import java.util.List;
+
+import com.bakdata.conquery.apiv1.auth.PasswordCredential;
+import com.bakdata.conquery.io.xodus.MasterMetaStorage;
+import com.bakdata.conquery.models.auth.basic.LocalAuthenticationConfig;
+import com.bakdata.conquery.models.auth.basic.LocalAuthenticationRealm;
+import com.bakdata.conquery.models.auth.basic.TokenHandler;
+import com.bakdata.conquery.models.auth.basic.TokenHandler.JwtToken;
+import com.bakdata.conquery.models.auth.entities.User;
+import com.bakdata.conquery.models.config.ConqueryConfig;
+import com.bakdata.conquery.models.identifiable.ids.specific.UserId;
+import com.google.common.io.Files;
+import org.apache.commons.io.FileUtils;
+import org.apache.shiro.authc.AuthenticationException;
+import org.apache.shiro.authc.IncorrectCredentialsException;
+import org.apache.shiro.util.LifecycleUtils;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests the basic functionality of the LocalAuthenticationRealm, which stores
+ * the credential data in a local database and offers simple user management
+ * (add/update/delete).
+ */
+public class LocalAuthRealmTest {
+
+	@Test
+	public void test() {
```

Comment:
Du könntest die tests die du hier machst einzeln machen und den Kontext in @BeforeAll bereitstellen

Reply:
Die for-Schleife über die Token ist um das try-catch. Generell sollte bei einem validen Token aber keine Exception geworfen werden.

Reply:
Done ;)